charts/pega/values-large.yaml

```diff
@@ -101,9 +101,11 @@ global:
           # For GKE to use static IP for load balancer, use kubernetes.io/ingress.global-static-ip-name: <global-static-ip-name>
           ssl_annotation:
 
+
       replicas: 1
+      # Pass the cosmosUI url as a Dynamic System Setting to the infinity. Added only when Constellation is enabled.
+      cosmosUISettings: "Pega-UIEngine/CosmosServicesURI=/prweb/cosmosUI"
```

Comment:
We may not even need this here.  Not sure why we wouldn't just insert the DSS when cosmosui.enabled is true.

Reply:
@dcasavant We want the customer to have the ability to change the cosmosservicesURI to a url of his choice. So we provided the value in values.yaml as well. If he does not change then we will always default to this.

Reply:
We disccused this with Gopal and added a new entry in the pega docker image as well to be able to pass this

https://github.com/pegasystems/docker-pega-web-ready/pull/74/files

From our last discussion we gathered that if we used NODE_SETTINGS param then we would override things, so we got rid of that and created a parameter to be able to pass this DSS. And to keep this customizable at the customer end we added the same in values.yaml as well.

Reply:
`Pega-UIEngine/CosmosServicesURI` will never change, will it?  Why include it here?  Also `/prweb` is configured elsewhere so that should be taken from the setting @pega-embate is adding.  I think all you need is `/cosmosUI`, which can be defaulted to that value and only specified if a client wants to change it.  

Maybe change it to `cosmosPath: "/cosmosUI"`?  (and defaulted to that value)